test/e2e/kubernetes/pod/pod.go

```diff
@@ -691,19 +691,18 @@ func (p *Pod) CheckLinuxOutboundConnection(sleep, duration time.Duration) (bool,
 					}
 					installedCurl = true
 				}
-				// if we can curl bing.com we have outbound internet access
-				out, err := p.Exec("--", "curl", "bing.com")
-				if err == nil {
-					readyCh <- true
-				} else {
-					// in case bing.com is down let's hope google.com is also not down
-					_, err := p.Exec("--", "curl", "google.com")
+				// if we can curl an external URL we have outbound internet access
+				urls := getExternalURLs()
+				for i, url := range urls {
+					out, err := p.Exec("--", "curl", url)
 					if err == nil {
 						readyCh <- true
 					} else {
-						// if both bing.com and google.com are down let's say we don't have outbound internet access
-						log.Printf("Error:%s\n", err)
-						log.Printf("Out:%s\n", out)
+						if i > (len(urls) - 1) {
```

Comment:
this will never be true, i will never be > len of the array - 1.

it should be `==`